plugins/mysql/mysql_connections_per_user

```diff
@@ -100,6 +100,9 @@ sub print_graph_data() {
         if($print_user eq "root") {
             $print_user = "root_";
         }
+        if (  $print_user =~ /[^A-Za-z0-9]/ ) {
+            $print_user =~ s/[^A-Za-z0-9]/_/;
+        }
```

Comment:
Please use the perl module "Munin::Plugin" for generating a proper fieldname according to munin's restrictions. Just use `clean_fieldname($print_user)`. This function also handles the "root" exception.
Thus you can avoid duplicate code.